tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -25,6 +25,10 @@ for allowed_hypervisor_annotation in "${non_formatted_allowed_hypervisor_annotat
 done
 allowed_hypervisor_annotations=$(echo $allowed_hypervisor_annotations | sed 's/,$//')
 
+SNAPSHOTTER_HANDLER_MAPPING="${SNAPSHOTTER_HANDLER_MAPPING:=}"
```

Comment:
Based on https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html I think this is supposed to be `{SNAPSHOTTER_HANDLER_MAPPING:-}` as we are effectively double assigning here? I guess it will still work though?